amundsen_application/static/css/_layouts.scss

```diff
@@ -5,6 +5,8 @@
 
 $resource-header-height: 84px;
 $aside-separation-space: 40px;
+$screen-lg-max: 1490px;
+$screen-lg-container: 1440px;
```

Comment:
Assuming this sizing came from Design or that they will verify it.

Reply:
Yep, it seems they have been always working with that size: https://www.figma.com/file/pylWNZTjyEOO74ZEmDV4f6/Master---Screens-%26-UI-Library-%5BAmundsen%5D?node-id=0%3A3433